.taskcluster.yml

```diff
@@ -135,3 +140,39 @@ tasks:
           description: Taskcluster boot utilities - push docker image
           owner: bastien@mozilla.com
           source: https://github.com/mozilla/task-boot
+
+    - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+      then:
+        taskId: {$eval: as_slugid("release")}
+        dependencies:
+          - {$eval: as_slugid("docker_push")}
+        provisionerId: proj-relman
+        workerType: ci
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '1 hour'}
+        extra:
+          github:
+            events:
+              - push
```

Comment:
As discussed on IRC, let's remove the extra since it is unneeded.

Reply:
Removed, and i pushed a new tag to trigger all the tasks